telemetry/metrics.go

```diff
@@ -23,6 +25,11 @@ const (
 	FormatText       = "text"
 )
 
+// DisplayableSink is an interface that defines a method for displaying metrics.
+type DisplayableSink interface {
+	DisplayMetrics(resp http.ResponseWriter, req *http.Request) (interface{}, error)
```

Comment:
nit: just for brevity

```suggestion
	DisplayMetrics(resp http.ResponseWriter, req *http.Request) (any, error)
```